packages/app-project/src/shared/components/Stat/components/AnimatedNumber/AnimatedNumber.spec.js

```diff
@@ -1,10 +1,41 @@
-import { render } from 'enzyme'
+import { mount, render } from 'enzyme'
 import React from 'react'
+import sinon from 'sinon'
 
 import AnimatedNumber from './AnimatedNumber'
 
+const DURATION = 1000
+const FORMATTED_VALUE = '1,000'
+const NEW_FORMATTED_VALUE = '2,000'
+const NEW_VALUE = 2000
+const VALUE = 1000
+
 describe('Component > AnimatedNumber', function () {
   it('should render without crashing', function () {
     render(<AnimatedNumber value={1000} />)
   })
+
+  it('should render the `value` by default', function () {
+    const wrapper = render(<AnimatedNumber duration={DURATION} value={VALUE} />)
+    expect(wrapper.text()).to.equal(FORMATTED_VALUE)
+  })
+
+  describe('animation behaviour', function () {
+    it('should animate from zero on mount', function () {
+      const animateNumberSpy = sinon.spy(AnimatedNumber.prototype, 'animateValue')
```

Comment:
You should call `animateNumberSpy.restore()` after the expectation runs, else this stays wrapped while the tests are run.